test/functional/interface_zmq.py

```diff
@@ -40,6 +40,13 @@ def set_test_params(self):
     def setup_nodes(self):
         skip_if_no_py3_zmq()
         skip_if_no_bitcoind_zmq(self)
+
+        # Import keys
```

Comment:
Why doesn't the default framework `import_deterministic_coinbase_privkeys` call work in this test? There should be a comment here to explain why this override is needed if the default behavior can't be fixed.